Cargo.toml

```diff
@@ -23,6 +23,7 @@ anyhow = "1.0.68"
 heck = "0.4.0"
 typed-builder = "0.18.0"
 embedded-can = "0.4.1"
+defmt = "0.3.8"
```

Comment:
```suggestion
```

This doesn't need to be there. The dependency is not used in the code, and having it there prevents compilation on windows

Reply:
Should be fixed now, thanks!